spec/controllers/statistics_controller_spec.rb

```diff
@@ -45,7 +45,7 @@
     it 'populates age_counts and gender_counts variables' do
       get :demographics
 
-      expect(assigns(:age_counts)).to eq({ "35-44" => 1 })
+      expect(assigns(:age_counts)).to eq({ "45-54" => 1 })
```

Comment:
Maybe instead of harcoding this we could create the member using something like `date_of_birth: Time.now - 46.years`